### PR TITLE
Add Engine DJ.app v3.0.1 cask engine-dj.rb

### PR DESCRIPTION
**Pull request by djasonic**
Application formerly known as Engine Prime cask engine-prime.rb

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/engine-dj.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+cask "engine-dj" do
+  version "3.0.1,6c40df4446"
+  sha256 "f4f7e212d2b3bf47087ef2caa9ee710559584ccdca680dd860ed387a3aea6a31"
+
+  url "https://imb-cicd-public.s3.amazonaws.com/Engine/#{version.csv.first}/Release/EDJ/mac/5e1011cdeb/Engine_DJ_Release_#{version.csv.first}_#{version.csv.second}.dmg",
+      verified: "imb-cicd-public.s3.amazonaws.com/Engine/"
+  name "Engine DJ Desktop"
+  desc "World's premier DJ software suite"
```
Comment (bevanjkay) [inline Casks/engine-dj.rb:8]: Please update the `desc` to fit the [desc stanza guidelines in the Cask Cookbook](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

> Do not use empty marketing jargon.

**Review by bevanjkay — DISMISSED** [review]
From CI audit:
```
Some packages are still installed, add them to uninstall pkgutil: com.airmusictechnology.engineprime.application
```

**Comment by razvanazamfirei** [comment]
> Application formerly known as Engine Prime cask engine-prime.rb

At this point I wonder if `engine-prime` should be removed
```
engine-prime
30 days: n/a
90 days: n/a
365 days: 13 (#5499)
Age: 1354 days (added 2019, September 13)
```

File: Casks/engine-dj.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  pkg "Engine DJ_#{version.csv.first}_Setup.pkg"
+
+  uninstall pkgutil: [
+    "com.airmusictechnology.enginedj.application",
+    "com.airmusictechnology.engineprime.application",
+  ]
+
+  zap trash: [
+    "~/Library/Application Support/AIR Music Technology",
+    "~/Library/Caches/AIR Music Technology",
```
Comment (bevanjkay) [inline Casks/engine-dj.rb:28]: @razvanazamfirei I wonder if these folders are specific to this application or if they are used as common folders between the vendor's applications. This is always a tricky one with `zap` stanzas.

Comment (razvanazamfirei) [reply Casks/engine-dj.rb]: Good point! Maybe something like my last change?

```
zap trash: [
        "~/Library/Application Support/AIR Music Technology/EnginePrime",
        "~/Library/Caches/AIR Music Technology/EnginePrime",
        "~/Library/Preferences/com.air-music-technology.EnginePrime.plist",
        "~/Library/Saved Application State/com.air-music-technology.EnginePrime.savedState",
        "~/Music/Engine Library",
      ],
      rmdir: [
        "~/Library/Application Support/AIR Music Technology",
        "~/Library/Caches/AIR Music Technology",
      ]
```

**Comment by miccal** [comment]
Thank you @djasonic.

**Review by miccal — APPROVED** [review]

**Pull request merged**